man.py

```diff
@@ -6,9 +6,12 @@
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "volunteer_planner.settings.local")
     os.environ.setdefault("DATABASE_PW", "")
     os.environ.setdefault("DB_USER", "")
+    os.environ.setdefault("SECRET_KEY", "1")
+    os.environ.setdefault("LANGUAGE", "de")
```

Comment:
The reason to have it also here, is to be able to change it (en/de) it without changing the settings file.